libraries/AP_HAL_ChibiOS/hwdef/CubeOrange/hwdef-bl.dat

```diff
@@ -29,7 +29,21 @@ FLASH_BOOTLOADER_LOAD_KB 128
 define HAL_LED_ON 0
 
 # order of UARTs (and USB)
-SERIAL_ORDER OTG1 UART7
+SERIAL_ORDER OTG1 USART2 USART3 UART7
+
+# Another USART, this one for telem1. This one has RTS and CTS lines.
```

Comment:
USART ??? shoudn't it be UART?

Reply:
Not according to the CubeOrange hwdef.dat file:
https://github.com/ArduPilot/ardupilot/blob/8544d00c97957ae2c2e28c0ceb15ca23e89d86b2/libraries/AP_HAL_ChibiOS/hwdef/CubeOrange/hwdef.dat#L32